From 28b2ce05cbdac453fee235983d090783313d0111 Mon Sep 17 00:00:00 2001
From: Henrique Gontijo <hgontijo@guidewire.com>
Date: Sun, 12 May 2019 17:11:02 -0700
Subject: [PATCH 491/806] configs: Enable PIDs cgroup

My use case to is to allow Kubernetes master to run on Raspberry Pi 3.
Kubernetes introduced [Pid limiting](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190129-pid-limiting.md),
on v.1.14.0 which includes [Pod Pids Limit](https://github.com/kubernetes/kubernetes/commit/bce9d5f2043bd86964c9fec80d466e47776071bc)
and [Node Pids Limit](https://github.com/kubernetes/kubernetes/commit/2597a1d97ef4d8f54b1ca661453e32794b756909#diff-fa76bb6ae2d9b4bb3d023737fe5e6029R333)
that requires pids cgroup.

See: https://github.com/raspberrypi/linux/pull/2968
---
 arch/arm/configs/bcm2709_defconfig | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -20,6 +20,7 @@ CONFIG_CGROUP_FREEZER=y
 CONFIG_CPUSETS=y
 CONFIG_CGROUP_DEVICE=y
 CONFIG_CGROUP_CPUACCT=y
+CONFIG_CGROUP_PIDS=y
 CONFIG_NAMESPACES=y
 CONFIG_USER_NS=y
 CONFIG_SCHED_AUTOGROUP=y
